Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -328,15 +329,13 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
         let cell = tableView.dequeueReusableCell(withIdentifier: String(describing: PhotonCustomViewCell.self), for: indexPath) as! PhotonCustomViewCell
         cell.tintColor = self.tintColor
         cell.customView = action.customView
-        cell.onSizeChange = {
+        cell.onSizeChange = { [weak self] in
             CATransaction.begin()
-            self.tableView.beginUpdates()
             CATransaction.setCompletionBlock {
-                UIView.animate(withDuration: 5) {
```

Comment:
Does it still resize with animation?

Reply:
Yes. It hasn't effect on animation. 
FYI: The animation is taking less then 1 second but we set it 5 second.